Make.config

```diff
@@ -273,6 +273,12 @@ DOTNET_MIN_TVOS_SDK_VERSION=11.0
 DOTNET_MIN_MACCATALYST_SDK_VERSION=13.1
 DOTNET_MIN_MACOS_SDK_VERSION=10.15
 
+# Minimum OS versions when using NativeOAT - these are at least the general min versions above (but may be higher).
+DOTNET_MIN_NATIVEAOT_IOS_SDK_VERSION=12.2
+DOTNET_MIN_NATIVEAOT_TVOS_SDK_VERSION=12.2
+DOTNET_MIN_NATIVEAOT_MACCATALYST_SDK_VERSION=13.1
+DOTNET_MIN_NATIVEAOT_MACOS_SDK_VERSION=10.15
```

Comment:
Do we still need this? (Not sure which dotnet/runtime -> dotnet/sdk is ingested at the moment but the Swift workaround should not be necessary anymore.)

Reply:
@filipnavara do you know the PR/commit that fixed it?

Reply:
https://github.com/dotnet/runtime/commit/f42efaa182270c101e9a39d0ffb38357c8bfbcb5

Reply:
Filed to look at it after this PR is in: https://github.com/xamarin/xamarin-macios/issues/18477